dub.json

```diff
@@ -11,7 +11,7 @@
     "stringImportPaths" : [ "build" ],
     "excludedSourceFiles": [ "source/scpp/*.d", "source/scpd/quorum/QuorumIntersectionTests.d" ],
     "preGenerateCommands": [
-        "$DUB source/agora/cli/version/main.d",
+        "$DUB source/agora/cli/version/main.d || sleep 5s || $DUB source/agora/cli/version/main.d",
```

Comment:
why do we have || instead of &&? 
I think whatever prints out 'Failed to execute' will return with an error code of non-zero

Reply:
Indeed, good catch!